EKF/ekf_helper.cpp

```diff
@@ -125,6 +155,20 @@ bool Ekf::resetPosition()
 		// use EV accuracy to reset variances
 		setDiag(P,7,8,sq(_ev_sample_delayed.posErr));
 
+	} else if (_control_status.flags.opt_flow) {
+		if (!_control_status.flags.in_air) {
+			// we are likely starting OF for the first time so reset the horizontal position and vertical velocity states
```

Comment:
Comment is inconsistent with what is done below. Where are we resetting vertical velocity?

Reply:
yes, I found that when horizontal velocity is diverged, the vertical velocity is also not correct sometimes.